packages/storybook/stories/CallComposite/CallCompositeDocs.tsx

```diff
@@ -182,6 +182,26 @@ export const getDocs: () => JSX.Element = () => {
         accepts an object where you can provide a custom JSX element against an icon name key.
       </Description>
 
+      <Heading>Customize Call Controls</Heading>
```

Comment:
This is great! Should we also add to the callwithchat composite docs also?

Reply:
maybe just a extension about the two extra buttons linking back to this page?

Reply:
Don e